tianshou/data/batch.py

```diff
@@ -511,12 +518,14 @@ def cat_(self, batch: 'Batch') -> None:
                 raise TypeError(s)
 
     @staticmethod
-    def cat(batches: List['Batch']) -> 'Batch':
-        """Concatenate a :class:`~tianshou.data.Batch` object into a single
+    def cat(batches: List[Union[dict, 'Batch']]) -> 'Batch':
```

Comment:
Same, is there a usecase for adding `dict` as supported input type ?

Reply:
To keep consistent with ``stack``.